src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -105,6 +106,13 @@
     String getWorldName();
 
     /**
+     * Gets the directory the files for this world are located.
+     *
+     * @return The directory the files are located in
+     */
+    File getWorldDirectory();
+
+    /**
```

Comment:
@modwizcode yeah but that's only in Sponge's implementation. So far the API has not defined how worlds should be stored so some implementation 'MinecraftAsSQL' may not want to store it in a file system. Just a thought.